.coveragerc

```diff
@@ -0,0 +1,25 @@
+[run]
+# measure 'branch' coverage in addition to 'statement' coverage
+# See: http://coverage.readthedocs.io/en/coverage-4.5.1/branch.html
+branch = True
+
+# list of directories or packages to measure
+source = fontParts
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # keywords to use in inline comments to skip coverage
+    pragma: no cover
+
+    # don't complain if tests don't hit defensive assertion code
+    raiseNotImplementedError
```

Comment:
Typo?

Reply:
No, not a typo. That's how it's used in many files. The actual lines are all like `self.raiseNotImplementedError`.

Reply:
Oh that’s a fontparts thing. Ok then